iked/vroute.c

```diff
@@ -602,6 +624,13 @@ vroute_dodefaultroute(struct iked *env, int flags, int addrs, uint8_t type,
 	default:
 		return (-1);
 	}
+
+	if (type == RTM_ADD)
```

Comment:
Where does this come from? Was it missing before? Same question for the block below.